src/encoded/audit/experiment.py

```diff
@@ -4283,6 +4283,36 @@ def check_experiment_atac_encode4_qc_standards(experiment, files_structure):
                     yield AuditFailure('mild to moderate bottlenecking', pbc2_detail, level='WARNING')
 
 
+def audit_analysis_files(value, system, excluded_types):
+    if 'analysis_objects' not in value:
+        return
+    experiment_files = (
+        f['@id']
+        for f in (
```

Comment:
I think that all the files are traversed in "def create_files_mapping(files_list, excluded):"  would not it be more efficient and clear to create another group of files there - and then use this group (experiment_files) here? Another question is about contributing files, I thought Analysis will only include the output files, not controls/references?